wikipedia/track.py

```diff
@@ -0,0 +1,108 @@
+import csv
+import math
+import random
+import re
+from os import getcwd
+from os.path import dirname
+from typing import Iterator
+
+from esrally.track.params import ParamSource
+
+QUERIES_DIRNAME: str = dirname(__file__)
+QUERIES_FILENAME: str = f"{QUERIES_DIRNAME}/queries.csv"
+
+SEARCH_APPLICATION_ROOT_ENDPOINT: str = "/_application/search_application"
+
+
+def query_iterator(k: int) -> Iterator[str]:
+    with open(QUERIES_FILENAME) as queries_file:
+        csv_reader = csv.reader(queries_file)
+        next(csv_reader)
+        queries_with_probabilities = list(tuple(line) for line in csv_reader)
+
+        queries = [query for query, _ in queries_with_probabilities]
+        probabilities = [float(probability) for _, probability in queries_with_probabilities]
+
+        for query in random.choices(queries, weights=probabilities, k=k):
```

Comment:
I am concerned about repetability and consistency of results without predefining [random.seed](https://docs.python.org/3/library/random.html#random.seed).
In other tracks we have typically allowed [overriding the seed](https://github.com/elastic/rally-eventdata-track/blob/41cf1f94d955b7ac75454c0440b0b6ec610dee61/eventdata/parameter_sources/elasticlogs_bulk_source.py#L103).
Depending on what we need to achieve with this track, and whether it's important to have repeatable results with the same hardware (e.g. for nightly benchmarks) I'd recommend setting a default seed that can be overriden with a track parameter.

Reply:
Done. Thank you for the feedback.